Header header
time start_time
float32[] distributions
int32 T_len # length of each distribution
float32 period # distribution period in seconds
int32 t_cur # current time index
